props/blaster.h

```diff
@@ -274,7 +274,9 @@ class Blaster : public PROP_INHERIT_PREFIX PropBase {
   }
 
   // Make swings do nothing
-  void DoMotion(const Vec3& motion, bool clear) override {}
+  void DoMotion(const Vec3& motion, bool clear) override {
+    PropBase::DoMotion(motion, clear);
```

Comment:
This does nothing, it is exactly equivalent to not having a DoMotion() in this blaster prop at all.

Did you try replacing motion with "Vec3(0)" ? 
This would mean that DoMotion() would still be called, but with a zero vector, like i fyou were holding it perfectly still.

Reply:
ah. I do admit I'm unclear with how having `void DoMotion`  in 3 different files works, as far as which takes precedence etc....
but "Vec3(0)" does work, where "Vec()" did not.
Updated again.